TotalActivation/TotalActivation.py

```diff
@@ -10,33 +10,60 @@
 from TotalActivation.filters import hrf
 from TotalActivation.process.temporal import wiener, temporal_TA, mad
 from TotalActivation.process.spatial import tikhonov
+from TotalActivation.preprocess.input import load_nifti, load_nifti_nomask, load_matlab_data, load_text_data
 
 logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.DEBUG)
 
 __all__ = ["TotalActivation"]
 
 
 class TotalActivation(object):
-    def __init__(self):
+    def __init__(self, method_time='B', method_space='S', hrf='bold', Lambda=1 / 0.8095, cost_save = False):
         # Method_time: 'B', 'S' or 'W'
         # Method_space: 'S', 'T', None
         # HRF: 'bold', 'spmhrf'
-        self.config = {'Method_time': 'B',
-                       'Method_space': None,
-                       'HRF': 'bold',
-                       'Detrend': True,
-                       'Standardize': True,
-                       'Highpass': 0.01,
-                       'Lowpass': None,
-                       'TR': 2,
-                       'Lambda': 1 / 0.8095}
+
+        # Input parameters
+        self.method_time = method_time
+        self.method_space = method_space
+        self.hrf = hrf
+        self.Lambda = Lambda
+        self.cost_save = cost_save
+
+        # Empty fields
         self.data = None
         self.atlas = None
         self.deconvolved_ = None
-        self.n_voxels = 0
-        self.n_tp = 0
-        self.cost_save = False
-        self.masking = False
+
+    def _load_data(self, f, a=None, mask=True, type='nifti', detrend=True, standardize=True, highpass=0.01, lowpass=None,
+             TR=2):
+        """
+        Wrapper function for loading all kinds of data
+        
+        :return: data or data + atlas in 2D 
+        """
+
+        global cmd
+        if type is 'nifti':
```

Comment:
type is a python keyword, I would change name of this variable to something else like ftype